tracardi/process_engine/action/v1/notification_plugin_action.py

```diff
@@ -0,0 +1,152 @@
+                except JSONDecodeError:
+                    content = await response.text()
+
+                result = {
+                    "status": response.status,
+                    "content": content
+                }
+
+                if response.status in [200, 201, 202, 203]:
+                    return Result(port="response", value=result)
+                else:
+                    return Result(port="error", value=result)
+
+
+def register() -> Plugin:
+    return Plugin(
+        start=False,
+        spec=Spec(
+            module=__name__,
+            className='NotificationGeneratorAction',
+            inputs=['payload'],
+            outputs=['response', 'error'],
+            version="0.7.2",
+            license="MIT",
+            author="Mateusz Zitaruk",
+            init={
+                "source": {"id": "", "name": ""},
+                "template_name": None,
+                "subscriber_id": "profile@id",
+                "recipient_email": "profile@pii.email",
+
+                "payload": "{}"
+            },
+            manual="novu_plugin_action",
+            form=Form(
+                groups=[
+                    FormGroup(
+                        name="Novu notification plugin configuration",
+                        fields=[
+                            FormField(
+                                id="source",
+                                name="Resource",
+                                description="Select your api key resource containing your key from novu",
+                                component=FormComponent(type="resource", props={"label": "resource", "tag": "token"})
+                            ),
+                            FormField(
+                                id="template_name",
+                                name="Novu template name",
+                                description="Provide your template name.",
+                                component=FormComponent(type="text")
+                            ),
+                            FormField(
+                                id="subscriber_id",
+                                name="Recipient email address",
+                                description="Provide email address of this notification recipient.",
+                                component=FormComponent(type="dotPath")
+                            ),
+                            FormField(
+                                id="recipient_email",
+                                name="Subscriber ID",
+                                description="Provide subscriber ID",
+                                component=FormComponent(type="dotPath")
+                            )
+                        ]
+                    )
+                ]
+            )
+        ),
+        metadata=MetaData(
+            name="Novu trigger plugin",
+            desc="Create and send notification to chosen recipient.",
+            icon="globe",
```

Comment:
just make sure that we have such an icon. Maybe we should add Novu icon like for Matomo / Civi etc.

Reply:
Lets keep it globe. And I will check if they have novu icon.